test/items_test.rb

```diff
@@ -0,0 +1,34 @@
+require 'CSV'
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/items'
+
+
+class ItemTest < Minitest::Test
+
+  def setup
+    @data = Item.new({
+      :id => 1,
+      :name => "Bat",
+      :description => "long and slim",
+      :unit_price => 25,
+      :merchant_id => 0101,
+      :created_at => "AZ",
+      :updated_at => "CO"})
+
+      assert_instance_of Item, @data
+  end
+
+  def test_it_exists
+    @data
```

Comment:
I don't believe we need this lone `@data` 
Otherwise, it looks good!

Reply:
ok we can change that during the next pull request =)

Reply:
> ok we can change that during the next pull request =)

Sounds like a plan!